Client/Frontend/Settings/SettingsTableViewController.swift

```diff
@@ -364,10 +364,12 @@ class StringSetting: Setting, UITextFieldDelegate {
         cell.accessibilityTraits = UIAccessibilityTraits.none
         cell.contentView.addSubview(textField)
 
+        textField.font = DynamicFontHelper.defaultHelper.DefaultStandardFont
+
         textField.snp.makeConstraints { make in
             make.height.equalTo(44)
             make.trailing.equalTo(cell.contentView).offset(-Padding)
-            make.leading.equalTo(cell.contentView).offset(Padding)
+            make.leading.equalTo(cell.contentView).offset(Padding)        
```

Comment:
Do you mind reverting this spacing change?

Reply:
Apologies, completely overlooked the spacing. Reverted now